code-challenges.js

```diff
@@ -20,49 +20,86 @@ const fruit4 = "kiwi"
 
 // --------------------1) Create a function that takes in a number and determines if the given number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below.
 
-// Pseudo code:
+// Pseudo code: I will input the variable of the boilingpoint water into an if and else if statement to return the following statements depending on the water temperature.
+// My Code #1 
+const boilingPoint = (water) => {
+    if (temp1 > 212) { return "is above boiling point"
+   } 
+     else if (temp1 < 212) {return `${temp1} is below boiling point`} 
+     else if (temp1 == 212){return "is at boiling point"}
+     else return "Check water again"
+   }
+   
+   const temp1 = 42
+   console.log(boilingPoint(temp1))
 
-const temp1 = 42
 // Expected output: "42 is below boiling point"
-
+// My Code #2
+const boiling= (water) => {
+    if (temp2 > 212){return `${temp2} is above boiling point`}
+     else if  (temp2 < 212){return `${temp2} is below boiling point`} 
+     else if (temp2 == 212){return `${temp2} is at boiling point`}
+     else return "Check water again"
+}
 const temp2 = 350
+console.log(boiling(temp2))
 // Expected output: "350 is above boiling point"
-
+// My Code #3
```

Comment:
Remember that we can use a function multiple times. You can refactor this and make it dynamic. You are definitely on the right path by using water as a parameter!